news/_posts/core-weekly/2022-08-08-coreweekly-31-2022.md

```diff
@@ -0,0 +1,147 @@
+* [#1378](https://github.com/PrestaShop/docs/pull/1378): Link for Clone the repository using Git. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+
+
+### ps-monitor-module-releases
+* [#7](https://github.com/PrestaShop/ps-monitor-module-releases/pull/7): Re-compute page every day by GitHub Action, by [@matks](https://github.com/matks)
+
+
+### Category tree links module
+* [#61](https://github.com/PrestaShop/ps_categorytree/pull/61): Bump version to 2.0.3. Thank you [@okom3pom](https://github.com/okom3pom)
+* [#59](https://github.com/PrestaShop/ps_categorytree/pull/59): Fix the category tree on pages that are not a category, by [@kpodemski](https://github.com/kpodemski)
+
+
+### Customer reassurance block module
+* [#435](https://github.com/PrestaShop/blockreassurance/pull/435): Bump @babel/core from 7.18.9 to 7.18.10. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#434](https://github.com/PrestaShop/blockreassurance/pull/434): Bump eslint from 8.20.0 to 8.21.0. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### Faceted search module
+* [#697](https://github.com/PrestaShop/ps_facetedsearch/pull/697): Bump eslint from 8.20.0 to 8.21.0. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#696](https://github.com/PrestaShop/ps_facetedsearch/pull/696): Bump @babel/cli from 7.18.9 to 7.18.10. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#695](https://github.com/PrestaShop/ps_facetedsearch/pull/695): Bump @babel/node from 7.18.9 to 7.18.10. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#694](https://github.com/PrestaShop/ps_facetedsearch/pull/694): Bump @babel/core from 7.18.9 to 7.18.10. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#693](https://github.com/PrestaShop/ps_facetedsearch/pull/693): Bump @babel/preset-env from 7.18.9 to 7.18.10. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#679](https://github.com/PrestaShop/ps_facetedsearch/pull/679): fix FacetFilters widget disappears after a no facet filter. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+
+
+### Contextual Help API
+* [#13](https://github.com/PrestaShop/contextual-help-api/pull/13): Use GitHub as source instead of confluence, by [@atomiix](https://github.com/atomiix)
+* [#10](https://github.com/PrestaShop/contextual-help-api/pull/10): Launching production environment. Thank you [@djodjo3](https://github.com/djodjo3)
+
+
+### PrestaShop contributors website
+* [#85](https://github.com/PrestaShop/TopContributors/pull/85): Migrate from Jenkins to Github Pages, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Hummingbird theme
+* [#358](https://github.com/PrestaShop/hummingbird/pull/358): return table main checkbox fix. Thank you [@JBLach](https://github.com/JBLach)
+
+
+### ps_distributionapiclient
```

Comment:
Need to modify the repository name for something more meaningful

And then need to modify https://github.com/PrestaShop/core-weekly-generator/blob/master/coreteam.py mapping file so that it does not need to be done every week